app/src/main/res/values-qq/strings.xml

```diff
@@ -1712,4 +1712,37 @@
   <string name="recommended_content_survey_dialog_message">Message text for the Recommended Content survey dialog.</string>
   <string name="recommended_content_survey_dialog_input_hint">Hint label for the Recommended Content survey dialog input field.</string>
   <string name="recommended_content_survey_dialog_submitted_message">Snackbar message for the Recommended Content survey after submitting it.</string>
+  <string name="donor_history_title">Title for the Donor history screen.</string>
+  <string name="donor_history_save_button">Button label for the Donor history screen to save the information.</string>
+  <string name="donor_history_update_donor_status_default">Label of the update donor status button.</string>
+  <string name="donor_history_update_donor_status_donor">Dialog option for the donor status that indicates as a donor.</string>
+  <string name="donor_history_update_donor_status_not_a_donor">Dialog option for the donors status that indicates as not a donor.</string>
+  <string name="donor_history_last_donated">Label of the last donated field.</string>
```

Comment:
Sorry, I know it's a pain, but these `qq` descriptions need some more work. (otherwise we will hear from Amir!)

Without having any other context, a phrase like "Label of the last donated field" has no meaning. It should be something like "Label that indicates the last date on which the user made a donation."

Instead of "Label of the recurring donor", it can be something like "Label indicating that the user is a recurring donor", and so on.

Reply:
Sure, I thought about the same thing but did not make proper descriptions. Done!